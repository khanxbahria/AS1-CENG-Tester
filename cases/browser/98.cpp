#include "../../your_code/Window.cpp"
#include "../../your_code/Browser.cpp"
#include "../../your_code/LinkedList.h"
int main() {
{
Browser* brows = new Browser();
Window a;
Window b;
Window c;
Window d;
Tab A = Tab("fff", "ll","ttt");
Tab B = Tab("ss", "ii","g");
Tab C = Tab("aa", "s","p");
Tab D = Tab("yyy", "q","g");
Tab E = Tab("g", "hh","h");
Tab F = Tab("pp", "w","xx");
Tab G = Tab("r", "kk","e");
Tab H = Tab("ll", "dd","x");
Tab I = Tab("gg", "o","q");
Tab J = Tab("rrr", "o","qqq");
Tab K = Tab("nn", "ppp","ccc");
Tab L = Tab("qqq", "rrr","r");
Tab M = Tab("hh", "bb","ggg");
Tab N = Tab("p", "v","ttt");
Tab O = Tab("dd", "h","yy");
Tab P = Tab("kk", "ggg","ff");
Tab Q = Tab("iii", "ooo","b");
Tab R = Tab("dd", "t","w");
Tab S = Tab("ttt", "dd","ddd");
Tab T = Tab("bbb", "xx","ss");
Tab U = Tab("zzz", "c","nnn");
Tab V = Tab("b", "eee","xxx");
Tab W = Tab("r", "o","h");
Tab X = Tab("ggg", "a","nnn");
Tab Y = Tab("aa", "uu","pp");
Tab Z = Tab("d", "rrr","mm");
a.moveActiveTabTo(2);
b.newTab(A);
c.changeActiveTabTo(2);
d.newTab(B);
a.changeActiveTabTo(5);
b.moveActiveTabTo(5);
c.moveActiveTabTo(5);
d.newTab(C);
a.changeActiveTabTo(4);
b.changeActiveTabTo(4);
c.newTab(D);
d.moveActiveTabTo(4);
a.moveActiveTabTo(2);
b.newTab(E);
c.moveActiveTabTo(2);
d.moveActiveTabTo(2);
a.changeActiveTabTo(1);
b.newTab(F);
c.changeActiveTabTo(1);
d.moveActiveTabTo(1);
a.moveActiveTabTo(2);
b.changeActiveTabTo(2);
c.newTab(G);
d.changeActiveTabTo(2);
a.changeActiveTabTo(3);
b.changeActiveTabTo(3);
c.changeActiveTabTo(3);
d.moveActiveTabTo(3);
a.moveActiveTabTo(3);
b.newTab(H);
c.moveActiveTabTo(3);
d.newTab(I);
a.newTab(J);
b.newTab(K);
c.moveActiveTabTo(4);
d.newTab(L);
a.newTab(M);
b.changeActiveTabTo(4);
c.moveActiveTabTo(4);
d.moveActiveTabTo(4);
a.moveActiveTabTo(5);
b.changeActiveTabTo(5);
c.moveActiveTabTo(5);
d.changeActiveTabTo(5);
a.newTab(N);
b.newTab(O);
c.newTab(P);
d.moveActiveTabTo(4);
a.changeActiveTabTo(0);
b.moveActiveTabTo(0);
c.moveActiveTabTo(0);
d.moveActiveTabTo(0);
a.moveActiveTabTo(3);
b.newTab(Q);
c.moveActiveTabTo(3);
d.newTab(R);
a.newTab(S);
b.changeActiveTabTo(5);
c.newTab(T);
d.changeActiveTabTo(5);
a.newTab(U);
b.newTab(V);
c.newTab(W);
d.moveActiveTabTo(0);
a.changeActiveTabTo(1);
b.changeActiveTabTo(1);
c.newTab(X);
d.moveActiveTabTo(1);
a.newTab(Y);
b.newTab(Z);
c.newTab(Z);
d.changeActiveTabTo(1);
c.moveActiveTabTo(4);
a.moveActiveTabTo(1);
c.changeActiveTabTo(1);
a.moveActiveTabTo(4);
b.moveActiveTabTo(4);
c.moveActiveTabTo(4);
d.moveActiveTabTo(4);
a.changeActiveTabTo(4);
d.moveActiveTabTo(4);
a.changeActiveTabTo(3);
b.changeActiveTabTo(3);
d.changeActiveTabTo(3);
a.moveActiveTabTo(5);
b.changeActiveTabTo(5);
d.changeActiveTabTo(5);
a.changeActiveTabTo(0);
c.moveActiveTabTo(0);
a.moveActiveTabTo(2);
b.moveActiveTabTo(2);
c.changeActiveTabTo(2);
d.moveActiveTabTo(2);
c.moveActiveTabTo(1);
d.moveActiveTabTo(1);
b.changeActiveTabTo(4);
d.changeActiveTabTo(4);
a.changeActiveTabTo(5);
b.moveActiveTabTo(5);
c.changeActiveTabTo(5);
a.changeActiveTabTo(2);
b.moveActiveTabTo(2);
c.changeActiveTabTo(2);
d.changeActiveTabTo(2);
a.moveActiveTabTo(2);
c.changeActiveTabTo(2);
d.changeActiveTabTo(2);
b.changeActiveTabTo(5);
c.moveActiveTabTo(5);
a.moveActiveTabTo(5);
c.moveActiveTabTo(5);
a.changeActiveTabTo(4);
b.changeActiveTabTo(4);
c.moveActiveTabTo(4);
d.changeActiveTabTo(4);
a.changeActiveTabTo(4);
a.changeActiveTabTo(0);
b.moveActiveTabTo(0);
d.moveActiveTabTo(0);
a.moveActiveTabTo(5);
c.changeActiveTabTo(5);
d.changeActiveTabTo(5);
b.moveActiveTabTo(0);
c.changeActiveTabTo(0);
d.moveActiveTabTo(0);
a.changeActiveTabTo(5);
c.moveActiveTabTo(5);
d.moveActiveTabTo(5);
a.changeActiveTabTo(1);
a.moveActiveTabTo(3);
c.changeActiveTabTo(3);
d.changeActiveTabTo(3);
a.changeActiveTabTo(2);
c.changeActiveTabTo(2);
d.changeActiveTabTo(2);
a.moveActiveTabTo(3);
b.moveActiveTabTo(3);
c.changeActiveTabTo(3);
d.moveActiveTabTo(3);
a.moveActiveTabTo(2);
b.moveActiveTabTo(2);
c.changeActiveTabTo(2);
d.changeActiveTabTo(2);
a.changeActiveTabTo(4);
c.moveActiveTabTo(4);
a.changeActiveTabTo(5);
b.moveActiveTabTo(5);
d.moveActiveTabTo(5);
b.moveActiveTabTo(2);
a.changeActiveTabTo(1);
b.changeActiveTabTo(1);
c.changeActiveTabTo(1);
d.changeActiveTabTo(1);
a.moveActiveTabTo(5);
b.changeActiveTabTo(5);
c.moveActiveTabTo(5);
d.moveActiveTabTo(5);
a.moveActiveTabTo(2);
c.moveActiveTabTo(2);
d.moveActiveTabTo(2);
brows->addWindow(a);
brows->addWindow(b);
brows->addWindow(c);
brows->addWindow(d);
brows->newWindow();
brows->closeWindow();
brows->print();
brows->print();
brows->switchToWindow(8);
brows->getWindow(3);
brows->newWindow();
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->switchToWindow(5);
brows->closeWindow();
brows->print();
brows->switchToWindow(9);
brows->mergeAllWindows();
brows->closeWindow();
brows->closeEmptyWindows();
brows->newWindow();
brows->closeWindow();
brows->switchToWindow(8);
brows->switchToWindow(0);
brows->mergeAllWindows();
brows->switchToWindow(5);
brows->closeWindow();
brows->switchToWindow(0);
brows->newWindow();
brows->closeWindow();
brows->closeEmptyWindows();
brows->switchToWindow(9);
brows->mergeAllWindows();
brows->print();
brows->mergeAllWindows();
brows->getWindow(5);
brows->newWindow();
brows->closeWindow();
brows->closeEmptyWindows();
brows->newWindow();
brows->mergeAllWindows();
brows->getWindow(4);
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->newWindow();
brows->mergeAllWindows();
brows->print();
brows->closeWindow();
brows->print();
brows->mergeAllWindows();
brows->newWindow();
brows->closeEmptyWindows();
brows->switchToWindow(6);
brows->closeWindow();
brows->print();
brows->getWindow(3);
brows->switchToWindow(7);
brows->closeWindow();
brows->getWindow(0);
brows->getWindow(7);
brows->closeWindow();
brows->mergeAllWindows();
brows->switchToWindow(0);
brows->mergeAllWindows();
brows->closeWindow();
brows->mergeAllWindows();
brows->newWindow();
brows->getWindow(6);
brows->newWindow();
brows->switchToWindow(5);
brows->print();
brows->mergeAllWindows();
brows->print();
brows->mergeAllWindows();
brows->getWindow(5);
brows->closeWindow();
brows->newWindow();
brows->getWindow(3);
brows->newWindow();
brows->mergeAllWindows();
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->closeWindow();
brows->closeWindow();
brows->closeWindow();
brows->closeWindow();
brows->print();
brows->print();
brows->newWindow();
brows->print();
brows->newWindow();
brows->switchToWindow(0);
brows->mergeAllWindows();
brows->closeWindow();
brows->switchToWindow(1);
brows->switchToWindow(3);
brows->closeWindow();
brows->switchToWindow(5);
brows->closeEmptyWindows();
brows->print();
brows->mergeAllWindows();
brows->print();
brows->closeEmptyWindows();
brows->mergeAllWindows();
brows->getWindow(2);
brows->print();
brows->switchToWindow(9);
brows->closeWindow();
brows->closeWindow();
brows->print();
brows->print();
brows->mergeAllWindows();
brows->print();
brows->switchToWindow(0);
brows->mergeAllWindows();
brows->getWindow(0);
brows->newWindow();
delete brows;
}
	return 0;
}