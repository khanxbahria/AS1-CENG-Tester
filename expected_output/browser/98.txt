The active tab is tab 0




The active tab is tab 2
v
p
ttt





dd
ttt
ddd

uu
aa
pp

o
rrr
qqq

bb
hh
ggg

c
zzz
nnn

The active tab is tab 5
h
dd
yy

ooo
iii
b





ll
fff
ttt

hh
g
h

rrr
d
mm

w
pp
xx

eee
b
xxx

dd
ll
x

ppp
nn
ccc

The active tab is tab 2
kk
r
e





ggg
kk
ff

xx
bbb
ss

rrr
d
mm

q
yyy
g

o
r
h

a
ggg
nnn

The active tab is tab 2
s
aa
p

ii
ss
g

rrr
qqq
r





o
gg
q

t
dd
w

The active tab is tab 0




The active tab is tab 2
v
p
ttt





dd
ttt
ddd

uu
aa
pp

o
rrr
qqq

bb
hh
ggg

c
zzz
nnn

The active tab is tab 5
h
dd
yy

ooo
iii
b





ll
fff
ttt

hh
g
h

rrr
d
mm

w
pp
xx

eee
b
xxx

dd
ll
x

ppp
nn
ccc

The active tab is tab 2
kk
r
e





ggg
kk
ff

xx
bbb
ss

rrr
d
mm

q
yyy
g

o
r
h

a
ggg
nnn

The active tab is tab 2
s
aa
p

ii
ss
g

rrr
qqq
r





o
gg
q

t
dd
w

The browser is empty
The browser is empty
The active tab is tab 1








Window is empty
The list is empty.
Window is empty
The list is empty.
The browser is empty
The active tab is tab 0




The active tab is tab 0




The active tab is tab 1








Window is empty
The list is empty.
The browser is empty
The browser is empty
The active tab is tab 0




The browser is empty
The browser is empty
The browser is empty
The browser is empty
The browser is empty
The browser is empty
